src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -156,7 +156,7 @@ bool emberAfGeneralCommissioningClusterArmFailSafeCallback(app::CommandHandler *
                                                            const Commands::ArmFailSafe::DecodableType & commandData)
 {
     MATTER_TRACE_EVENT_SCOPE("ArmFailSafe", "GeneralCommissioning");
-    MATTER_TRACE_SCOPE(::chip::Tracing::Scope::GeneralCommissioning_ArmFailSafe);
+    MATTER_TRACE_SCOPE("ArmFailSafe", "GeneralCommissioning");
```

Comment:
Seems like it should be possible to transition these without duplicating. Thoughts?

Reply:
My plan was/is to fully replace it once I have functional equivalence. I found not users of pw_trace, but I did find users of perfetto, so I was planning to switch over when the perfetto backend is included and functional.